psql2mysql.py

```diff
@@ -181,10 +189,10 @@ def migrate(self):
         target_tables = self.target_db.getTables()
 
         if not source_tables:
```

Comment:
Shouldn't this check be part of `precheck` as well?

Reply:
What should be the outcome there? It's not exactly an error (not even handled as that in the migrate phrase). Just print a warning?

Reply:
I do not have the setup to test right now. So how does it behave when you run a precheck on empty table? If it does not print an ugly error, I'm fine with it.

Reply:
I mean - we do not have to warn during precheck that there's an empty table that should not be migrated. My only concern is that passing empty table to precheck does not break

Reply:
OK, if precheck does fine, that's good then.